ResearchKit/Consent/ORKConsentLearnMoreViewController.m

```diff
@@ -54,11 +54,9 @@ - (instancetype)initWithHTMLContent:(NSString *)content {
     return self;
 }
 
--(instancetype)initWithContentURL:(NSURL * __nonnull)contentURL
-{
+- (instancetype)initWithContentURL:(NSURL * __nonnull)contentURL {
     self = [super init];
-    if(self)
-    {
+    if(self) {
```

Comment:
Missing space after `if`. You could also fix this everywhere: a simple search-and-replace will do.